regulations/html.py

```diff
@@ -56,6 +52,19 @@ def __init__(self, language, buildDir, pdfName, isTranslation=False, verbose=Fal
     if not self.isTranslation:
       self.pages()
 
+  def __del__(self):
+    self.regulations_fp.close()
+
+  def find_matching_reg(self, reg_id):
+    # Go back to the beginning of regulations file
+    self.regulations_fp.seek(0)
```

Comment:
Thanks for porting this to python. Very smart of you to remember to close the file pointer, I totally would have forgotten that =). I'm not in love with us having to seek to the beginning of the file each time, however. Do you think it would make more sense to instead build up a list of regulations at the very start (in the constructor), and then we wouldn't need to hold onto a file pointer, nor would we have to repeatedly seek in a file?